### PR TITLE
Cherryupdate No-Intro DATs. 

### PR DESCRIPTION
**Pull request by lasers**
Added few updated No-Intro Dats from the site.
Added misplaced(?) Nintendo-NES.dat to rename cheat codes with. 

Thank you.

### CODE REVIEW

File: metadat/no-intro/Nintendo - Game Boy Advance.dat [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 )
 
 game (
+	name "[BIOS] Game Boy Advance (Japan) (Debug Version)"
+	description "[BIOS] Game Boy Advance (Japan) (Debug Version)"
+	rom ( name "[BIOS] Game Boy Advance (Japan) (Debug Version).gba" size 16384 crc 15E1F676 md5 E60E599135009129B288988A1CBA91DF sha1 AA98A2AD32B86106340665D1222D7D973A1361C7 )
+)
+
+game (
+	name "[BIOS] Game Boy Advance (World)"
```
Comment (RobLoach) [inline metadat/no-intro/Nintendo - Game Boy Advance.dat]: Mind removing the BIOS entries?

Comment (lasers) [inline metadat/no-intro/Nintendo - Game Boy Advance.dat]: I'm not sure if making a commit would automatically notify you. Is it fine now?

**Comment by RobLoach** [comment]
The Nintendo - Nintendo System Entertainment DAT is not there because it's in `dat/` instead. See the `header` skip at the top? It's there to skip some of the unneeded header information in the rom. RetroArch doesn't support header skips, so we put it at [`dat/Nintendo.dat`](https://github.com/libretro/libretro-database/blob/master/dat/Nintendo%20-%20Nintendo%20Entertainment%20System.dat) instead, with the header skip removed.

**Comment by lasers** [comment]
Okay. Now I see why that NES disappeared from that no-intro directory. I'm going to keep one there unmodified.

**Comment by lasers** [comment]
Go ahead and close if it is unwanted to leave an unmodified file inside no-intro directory. I re-read what you said and it now sounds like an unwanted behavior. Sorry. I only wanted to use the latest one for renaming. I went with gitignore.

**Comment by RobLoach** [comment]
A bunch of the NES hashes changed. Do you know if they the header information is different? Know where those changes came from?

**Comment by lasers** [comment]
No, sorry. It looks like there are bunch of updates on no-intro site for several dats. I can't say anything more than... I got them from http://datomatic.no-intro.org/?page=download 

I'm working on Game Boy cheats right now. Not easy splitting them up.

**Comment by RobLoach** [comment]
Just the NES changes aren't able to go in, since the hashes changed. Other then that, this is good! I can revert the NES changes if you don't get to them. Your call!

**Comment by lasers** [comment]
Go ahead and do it. I don't know git well enough. Also, I may need help with my `rename-gameboy` branch to get rid of unwanted commits before making a pull request. https://github.com/lasers/libretro-database

**Pull request merged**